src/app/CommandResponseSender.cpp

```diff
@@ -78,16 +76,33 @@ void CommandResponseSender::OnResponseTimeout(Messaging::ExchangeContext * apExc
 {
     ChipLogDetail(DataManagement, "CommandResponseSender: Timed out waiting for response from requester mState=[%10.10s]",
                   GetStateStr());
+    // We don't need to consider remaining async CommandHandlers here. We become the exchange
+    // delegate only after CommandHandler::OnDone is called.
```

Comment:
I don't see a CommandHandler::OnDone anywhere.  And there shouldn't be CommandHandlers (plural) involved anyway....

Trying to make sense of this comment and failing.

Reply:
Removed the comment.

I got worried about a potential corner case that is non-existent.